common/include/pcl/impl/point_types.hpp

```diff
@@ -172,8 +172,48 @@
   (pcl::BRISKSignature512)      \
   (pcl::Narf36)
 
+// Define all point types that have descriptorSize() member function
+#define PCL_DESCRIPTOR_FEATURE_POINT_TYPES \
+  (pcl::PFHSignature125)        \
+  (pcl::PFHRGBSignature250)     \
+  (pcl::FPFHSignature33)        \
+  (pcl::VFHSignature308)        \
+  (pcl::GASDSignature512)       \
+  (pcl::GASDSignature984)       \
+  (pcl::GASDSignature7992)      \
+  (pcl::GRSDSignature21)        \
+  (pcl::ESFSignature640)        \
+  (pcl::BRISKSignature512)      \
+  (pcl::Narf36)
+
+
 namespace pcl
 {
+  namespace traits
```

Comment:
this should reside somewhere else, of course

Reply:
if placed inside `pcl/traits.h` then there's a problem that point types have to be available there. a better way to put it into separate header. and though it seems like the best place, `pcl/point_traits.h` is deprecated. may i un-deprecate it?

Reply:
I think this is a good enough place for now